metapool/igm.py

```diff
@@ -0,0 +1,155 @@
+import os
+
+from datetime import datetime
+
+from openpyxl import load_workbook
+from openpyxl.styles import PatternFill
+
+_IGM_ROW_OFFSET = 25
+
+
+def _property_maker(obj, name, cell, default):
```

Comment:
Can _property_maker() be made a part of IGMManifest? IMHO, it seems unusual to have a helper function that begins with '_' outside of a class.

Reply:
Also, it may be good to add a description for the function/method.

Reply:
I added some documentation to this helper function - good call. This is is a "meta" 🧙  function that creates setters and getters for the `IGMManifest` class. I would prefer to keep separate from the class since it's not really something tha the IGMManifest class should know how to do. This is more of a convenience for us to not have to write or maintain 13 separate setters and getters.

Reply:
Fair point!